libtmux/common.py

```diff
@@ -270,16 +270,21 @@ def __init__(self, *args: t.Any, **kwargs: t.Any) -> None:
 
         stdout_str = console_to_str(stdout)
         stdout_split = stdout_str.split("\n")
-        stdout_filtered = list(filter(None, stdout_split))  # filter empty values
+        # remove trailing newlines from stdout
+        for v in reversed(stdout_split):
+            if v == '':
+                stdout_split.pop()
+            else:
+                break
```

Comment:
```suggestion
        while stdout_split and stdout_split[-1] == "":
            stdout_split.pop()
```

@rockandska Thoughts on this?

Reply:
Cleaner and does the job :+1: 

Could squash it to preserve clean history if needed